### DISCUSSION TITLE
whisper does not take into account the inserted filename

### DISCUSSION THREAD

**Discussion by CesiraBuffarlotti** [openai/whisper#2428] [General]
This is my code
```
model = whisper.load_model("base")
result = model.transcribe('/media/hdb1/snap.mp4')
print(result["text"])
```
in the same dir (/media/hdb1) there are several .mp4 files. Whisper works all of them and in result I get the transcription of snap.mp4 plus all other -mp4 files in the dir

**Comment by glangford** [comment]
You might try moving snap.mp4 to its own directory. Run whisper on it in that isolated directory and observe the results.

whisper doesn't process multiple files unless requested, and no bug has been reported where whisper does not respect the filename.

**Reply by CesiraBuffarlotti** [reply]
All downloaded files end up in a folder, and I don't want to move a file from one part to another every time. That doesn't sound right to me.

**Reply by glangford** [reply]
Moving the file is solely for test purposes.
